populator-machinery/controller.go

```diff
@@ -549,8 +675,40 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 	}
 
 	// *** Here is the first place we start to create/modify objects ***
+	// If the PVC is unbound and PVC' doesn't exist yet, create PVC'
+	if "" == pvc.Spec.VolumeName && pvcPrime == nil {
+		pvcPrime = &corev1.PersistentVolumeClaim{
```

Comment:
Same thing for PVC prime. If we add new fields to PVC, then we'll miss them here. Since this is already existing code, I'm fine if we address this in a followup.

Reply:
Added a TODO comment for later improvement